include/velocypack/Slice.h

```diff
@@ -621,45 +677,45 @@ class Slice {
     if (!isUTCDate()) {
       throw Exception(Exception::InvalidValueType, "Expecting type UTCDate");
     }
-    uint64_t v = readIntegerFixed<uint64_t, sizeof(uint64_t)>(_start + 1);
+    uint64_t v = readIntegerFixed<uint64_t, sizeof(uint64_t)>(valueStart() + 1);
     return toInt64(v);
   }
 
   // return the value for a String object
   char const* getString(ValueLength& length) const {
-    uint8_t const h = head();
+    uint8_t const h = valueHead();
     if (h >= 0x40 && h <= 0xbe) {
       // short UTF-8 String
       length = h - 0x40;
-      return reinterpret_cast<char const*>(_start + 1);
+      return reinterpret_cast<char const*>(valueStart() + 1);
     }
 
     if (h == 0xbf) {
       // long UTF-8 String
-      length = readIntegerFixed<ValueLength, 8>(_start + 1);
+      length = readIntegerFixed<ValueLength, 8>(valueStart() + 1);
       checkOverflow(length);
-      return reinterpret_cast<char const*>(_start + 1 + 8);
+      return reinterpret_cast<char const*>(valueStart() + 1 + 8);
```

Comment:
I think this should still operate on `start()` and not on `valueStart()`.
If a client application needs to peek into the value behind the tag, I think it should explicitly resolve the tag.